swagger/config.go

```diff
@@ -31,4 +33,6 @@ type Config struct {
 	PostBuildHandler PostBuildDeclarationMapFunc
 	// Swagger global info struct
 	Info Info
+	// [optional] If set, model builder should call this handler to get addition typename-to-swagger-format-field convertion.
+	AdditionalFormatHandler GetAdditionalSchemaFormat
```

Comment:
You are right. the whole "additional" prefix was not a good idea. Fixed. Does it make sense to send  new clean PR?